lib/options-manager.js

```diff
@@ -155,14 +156,32 @@ const normalizeOptions = options => {
 	return options;
 };
 
+const loadPackageConfig = options => {
+	const switchBackCwd = process.cwd();
+	const moduleName = 'xo';
+	// Yaml is disabled
+	const searchPlaces = ['package.json',
+		`.${moduleName}rc`,
+		`.${moduleName}rc.json`,
+		`.${moduleName}rc.js`,
+		`${moduleName}.config.js`];
+	const loaders = {noExt: defaultLoaders['.json']};
+	// Change working directory for load config
+	process.chdir(options.cwd);
```

Comment:
Instead of changing the working directory directory, pass `options.cwd` to comisconfig in option `searchFrom`

Reply:
Done. I didn't saw this option before.